networking_aci/plugins/ml2/drivers/mech_aci/cobra_manager.py

```diff
@@ -308,15 +310,34 @@ def ensure_subnet_created(self, network_id, address_scope_name, gateway):
                 # subnet_out_profile = RsBDSubnetToProfile(subnet,l3_out,tnL3extOutName=l3_out)
                 # self.apic.commit(subnet_out_profile)
 
-        self.apic.commit([subnet] + subnet_outs)
+        match_rule_prefixes = []
+        if network_az and scope_config.get('vrf'):
+            az_suffix = network_az[-1]
+            match_rule_dn = AZ_MATCH_RULE_DN.format(az=az_suffix.upper(), vrf=scope_config['vrf'].upper())
+            prefix = self._get_network(gateway)
```

Comment:
Shoudn't we clear the hostbits here?

Reply:
https://github.com/sapcc/networking-aci/pull/59/files#diff-72a59e8d5d40872d0f9faecfe9337811fba73da2ecc8c5860c79bf911f582447R698-R703

Disregard, you are cleaning hostbits

Reply:
As discussed in slack, `_get_network()` clears the hostbits.